holoviews/element/path.py

```diff
@@ -193,6 +193,8 @@ class Ellipse(BaseShape):
 
     aspect= param.Number(default=1.0, doc="The aspect ratio of the ellipse.")
 
+    angle = param.Number(default=0, doc="The angle in radian between the long axis and the horizontal.")
+
```

Comment:
To be compatible with our other packages, such a parameter should probably be called `orientation`, and the description could be "Orientation in the Cartesian coordinate system, with zero at 3 o'clock and increasing counterclockwise".